core/src/main/scala/net/ruippeixotog/scalascraper/browser/JsoupBrowser.scala

```diff
@@ -42,25 +47,25 @@ class JsoupBrowser(val userAgent: String = "jsoup/1.8", val proxy: JavaProxy = n
     executePipeline(Jsoup.connect(url).method(POST).proxy(proxy).data(form.asJava))
 
   def parseFile(file: File, charset: String): JsoupDocument =
-    JsoupDocument(Jsoup.parse(file, charset))
+    JsoupDocument(Jsoup.parse(file, charset, baseUri, parser))
```

Comment:
When `Jsoup.parse` is called with a file, the base URI applied is taken from the file.

```suggestion
    JsoupDocument(Jsoup.parse(file, charset, file.getAbsolutePath, parser))
```